webapp/main.py

```diff
@@ -129,53 +183,59 @@ def process_json(input_json=None):
         # entities -> url -> urls
 
         desc_urls = entities.get('description').get('urls')
-        if not desc_urls: continue 
+        if not desc_urls or len(desc_urls) == 0:
+            continue
```

Comment:
at first was worried len(e) in the second half of the `or` might throw an exception but looks like it's fine
```
>>> e = None
>>> if not e or len(e) == 0:
...     print("foo")
...
foo
```